provider/doc_edits_test.go

```diff
@@ -29,3 +29,44 @@ func TestEffectiveLabels(t *testing.T) {
 		})
 	}
 }
+
+func TestSecretsInPlainTextNote(t *testing.T) {
```

Comment:
```suggestion
func TestSecretsInPlainTextNote(t *testing.T) {
	t.Parallel()
```

Reply:
Isn't `t.Parallel()` in the loop below enough?

Reply:
They do different things:
1. `t.Parallel()` in the top level allows `TestSecretsInPlainTextNote` to run in the top level parallel pool with other parallel tests.
2. `t.Parallel()` nested allows those subtests to join the parallel pool for their supertest.

Skipping the top level parallel has each subtest of `TestSecretsInPlainTextNote` running concurrently while all other tests block.

I wasn't 100% sure, so I investigated [here](https://go.dev/play/p/LM5Og6ch9We).